kyo-core/shared/src/main/scala/kyo/KyoApp.scala

```diff
@@ -9,22 +9,13 @@ import scala.collection.mutable.ListBuffer
   *
   * Note: This class and its methods are unsafe and should only be used as the entrypoint of an application.
   */
-abstract class KyoApp extends KyoApp.Base[KyoApp.Effects]:
-    def log: Log.Unsafe = Log.unsafe
-    def random: Random  = Random.live
-    def clock: Clock    = Clock.live
```

Comment:
I'm working on new `Clock` and `Timer` implementations that will offer [APIs to control time](https://github.com/getkyo/kyo/blob/cb97b5d74131740572098d5ec3b611144acf4bde/kyo-core/shared/src/main/scala/kyo/Clock.scala#L132). Overriding the default services via these high-level combinators is a more user-friendly approach than having to implement them manually. `Random` and `Log` can already be overriden via their `let` method but I'll also look into providing convenient behavior customization APIs for them.